tensorflow_recommenders_addons/dynamic_embedding/__init__.py

```diff
@@ -14,6 +14,7 @@
 # ==============================================================================
```

Comment:
1. Plz update "How to compile GPU version" to README.md.
2. make workflow support GPU UnitTest.
3. You just only add a op, but not apply to TFRA?

Reply:
1. Accept.
2. Currently, there is no available GPU host for UnitTest.
3. Accept. The op is used in `de.embedding_lookup_sparse`.